### DISCUSSION TITLE
Teoría de Shannon

### DISCUSSION THREAD

**Discussion by eabol** [mmasias/progra2-22-23#1] [General]
Que ejemplos podríamos mencionar para ejemplificar la teoría de Shannon.

**Comment by Gbrl413** [comment]
el juego de mesa adivina quien

**Comment by miguelgomez75** [comment]
Para el juego ¿quien es? por ejemplo

**Comment by josueMerino** [comment]
Clasificación de plantas según su hoja

**Comment by sara-patino** [comment]
el juego ve a pescar

**Comment by Diegoza04** [comment]
akinator

**Comment by TheMoys** [comment]
Un juego de charadas.

**Comment by anibalbg** [comment]
Para el ahorcado

**Comment by danielpuentesaranana** [comment]
Para el ahorcado

**Comment by LavinAguado** [comment]
El juego quien es quien

**Comment by RaulPlayo** [comment]
Para el juego de mesa Adivina quien és

**Comment by AlejandroVinas** [comment]
El dibujar una imagen a pixeles usando un filtro que altera los colores de forma que el dibujante no sepa los colores exactos

**Comment by srgiom** [comment]
Para el ahorcado

**Comment by oscarsantasanchez** [comment]
Para el juego de quién es quién

**Comment by nekiiiiis** [comment]
quien es quien

**Comment by MikaOT** [comment]
Hundir la flota o battleship

**Comment by pablo4343434** [comment]
para el juego de 20 preguntas

**Comment by Dievex** [comment]
quien es quien

**Comment by paulaqing** [comment]
El juego de quién es quién

**Comment by Mariodrm17** [comment]
para el juego del ahorcado

**Comment by luciaZama** [comment]
Para jugar al ahorcado

**Comment by pablarce** [comment]
el akinator

**Comment by Dmon4516** [comment]
Para un estudio de población

**Comment by Jorgeog25** [comment]
Para jugar a "El ahorcado"

**Comment by VeronikaEspa** [comment]
El juego Clue

**Comment by daavidgutieerrez** [comment]
Para el quién es quién

**Comment by joserasilla-uneat** [comment]
Los juegos tipo Wordl

**Comment by rubensetien** [comment]
Para el jugar al veo veo

**Comment by DavidGarciaCosta** [comment]
El juego de hundir la flota

**Comment by 0xJVR** [comment]
Para saber la localización de algo o alguien.

**Comment by aadrigaar** [comment]
para jugar al ahorcado

**Comment by ikercelaya** [comment]
El juego de quién es quién

**Comment by AndrextA** [comment]
El juego de adivinar el personaje a partir de preguntas

**Comment by ceesargaarcia** [comment]
Para clasificar un grupo de personas en países y sexos (como por ejemplo en el grado de ingeniería informática)...

**Comment by CarlosCasadoo** [comment]
El juego wordle.

**Comment by pabloocanteroo** [comment]
cuando decides donde ir de vacaciones

**Comment by 31Diego** [comment]
Por ejemplo adivinar una carta en función de su color, tipo de carta, etc

**Comment by MRSergio21** [comment]
El juego para adivinar ¿Quien soy?

**Comment by edgarleon12** [comment]
Puedes usar este método para el juego de  "memoria" vas sacando pares mediante valores descubiertos anteriormente y haciendo preguntas tipo... Ya salió esta carta? En que columna, en qué posición.

**Comment by Jorge85015** [comment]
Convertir números de decimal a binario (dividir por 2) y depende a eso es 1 o 0
